dashboard/src/components/ChartView/ChartReadme.test.tsx

```diff
@@ -4,11 +4,11 @@ import * as ReactMarkdown from "react-markdown";
 
 import ChartReadme from "./ChartReadme";
 
-it("renders a loading message if the readme is not present", () => {
+it("renders a README not found message if the readme is not present", () => {
   const wrapper = shallow(
     <ChartReadme getChartReadme={jest.fn()} hasError={false} version="1.2.3" />,
   );
-  expect(wrapper.text()).toBe("Loading");
```

Comment:
Do we have any other test that targets the loading part of the component?

Reply:
I did follow a wrong approach for fixing this. The idea was to throw an error when the README is empty not remove the `Loading` message.